CMakeLists.txt

```diff
@@ -31,16 +31,20 @@ if (${STATIC_RUNTIME_LIBS})
     endif()
 endif()
 
-find_library(TURBOJPEG_LIBRARIES NAMES libturbojpeg.so.0 libturbojpeg turbojpeg PATHS "${PROJECT_SOURCE_DIR}/dependencies/lib")
-find_path(TURBOJPEG_INCLUDEDIR NAMES turbojpeg.h PATHS "${PROJECT_SOURCE_DIR}/dependencies/include")
-mark_as_advanced(TURBOJPEG_LIBRARIES TURBOJPEG_INCLUDEDIR)
-include_directories(${TURBOJPEG_INCLUDEDIR})
-link_directories(${TURBOJPEG_INCLUDEDIR})
-
 if (UNIX)
     include(FindPkgConfig)
 endif()
 
+if (UNIX)
+    pkg_check_modules(TURBOJPEG libjpeg)
```

Comment:
Do you mean libturbojpeg here? otherwise travis build is broken

Reply:
oh, right. forgot to push